_data/companies.csv

```diff
@@ -14,14 +16,20 @@ NamingMatters,http://www.namingmatters.com/,SaaS products,"Trademark pre-clearan
 PlaceOS,https://place.technology/,SaaS products,"Connects everything in your office, tenancy or building to create a seamless user experience"
 Rainforest QA,https://www.rainforestqa.com/,SaaS products,"QA at the Speed of Development, Intelligent Crowdtesting for Agile Teams"
 Void,https://voidapp.co/,SaaS products,"Read + Retain in a distracted world"
+Aveine,https://www.aveine.com/,Software Development,"The first smart wine aerator"
 Bulutfon,https://www.bulutfon.com/,Software Development,"Cloud Voice & VOIP solution for Turkey"
 Duo Design,https://www.duodesign.co.uk/,Software Development,"Web and application design and development from Manchester"
+Kemal,https://kemalcr.com/,Software Development,"Fast, Effective, Simple Web Framework"
+Kinko booking,https://kinkobooking.com,Software Development,"Gérez toutes vos Réservations en un Seul Click"
 Lechner Knowledge Center,https://lechnerkozpont.hu/en/oldal/cultural-heritage-protection,Software Development,"Cultural heritage inventory"
 Level UP Solutions,https://www.levups.com,Software Development,"Web application design and development, in France, specialized in e-commerce"
 LI-COR Biosciences,https://www.licor.com/,Software Development,"LI-COR Biosciences designs and manufactures instrument systems and research tools for environmental and biotechnology research"
+Lunar,https://lunar.fyi/,Software Development,"The defacto app for controlling monitors"
```

Comment:
The main product seems to be actually implemented in Swift. According to their submission, they're using Crystal for an adapter to control virtual displays over a network connection.
Not sure what's the best way to represent that here, but we should not insinuate the entire app is written in Crystal.

Reply:
I don't think the Crystal component is really a display driver, nor is it for macOS.
It seems to be a DDC-over-HTTP relay for Raspberry Pi?

Reply:
I think the main problem I have with this is that is suggests the entire app is written in Crystal, when it's actually just an adapter component.

Reply:
"but the UI" 🤣 

Now seriously, what can be an intermediate way to put it?

Reply:
Maybe?
```suggestion
Lunar,https://lunar.fyi/,Software Development,"Client adapter for monitor control over HTTP"
```